store/storetest/oauth_store.go

```diff
@@ -405,12 +406,17 @@ func testOAuthStoreDeleteApp(t *testing.T, ss store.Store) {
 		t.Fatal(err)
 	}
 
-	s1 := model.Session{}
+	s1 := &model.Session{}
 	s1.UserId = model.NewId()
 	s1.Token = model.NewId()
 	s1.IsOAuth = true
 
-	store.Must(ss.Session().Save(&s1))
+	s1, err := ss.Session().Save(s1)
```

Comment:
I would replace this entire block (and all that you have written to replace `store.Must`) as something like:
```go
s1, err := ss.Session().Save(s1)
require.Nil(t, err)
```